rvps/src/store/mod.rs

```diff
@@ -9,13 +9,16 @@ use crate::reference_value::ReferenceValue;
 
 use anyhow::Result;
 
+pub mod local_fs;
+
 /// Interface of a `Store`.
 /// We only provide a simple instance here which implements
 /// Store. In more scenarios, RV should be stored in persistent
 /// storage, like database, file and so on. All of the mentioned
 /// forms will have the same interface as following.
 pub trait Store {
-    /// Store a reference value
+    /// Store a reference value. If the given `name` exists,
+    /// return the previous `Some<ReferenceValue>`, or return `None`
```

Comment:
```suggestion
    /// Store a reference value. If the given `name` exists,
    /// return the previous `Some<ReferenceValue>`, otherwise return `None`
```

Reply:
Thanks, fixed it